packages/icondusk-e2e/package.py

```diff
@@ -9,33 +9,42 @@
 import subprocess
 import os
 
+
 class IconduskE2e(CMakePackage):
     """A library for numerical weather prediction and climate modelling"""
 
     homepage = 'https://github.com/dawn-ico/icondusk-e2e'
-    git      = 'git@github.com:dawn-ico/icondusk-e2e.git'
+    git = 'git@github.com:dawn-ico/icondusk-e2e.git'
     maintainers = ['cosunae']
 
     version('master', branch='master')
 
-    depends_on('atlas_utilities', type=('build','run'))
-    depends_on('dawn4py',  type=('build','run'))
+    depends_on('cmake@3.17.0:')
+    depends_on('boost@1.73.0%gcc@8.3.0')
```

Comment:
gcc@8.3.0 doesn't exit on Daint anymore, are you sure you specifically need this compiler?